src/components/atoms/DigitInput/DigitInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Input from '@material-ui/core/Input';
+import { withStyles } from '@material-ui/core/styles';
+
+const StyledDigitInput = withStyles({
+  root: {
+    width: 70,
+    height: 100,
+    border: '1px solid',
```

Comment:
이 부분은 꼭 바꿔야 하는 건 아니고 jikang님 의견을 들어보고 싶은데, border가 아예 black인 것보다는 색이 조금 연한 편이 숫자에 집중되지 않을까 싶기도 해서요. ㅎㅎ 어떻게 생각하시나요?
```suggestion
    border: '1px solid gray',
```
or
```suggestion
    border: '1px solid lightgray',
```

Reply:
solidgray가 더 좋은 방향인 것 같습니다. 제안 감사합니다!!